core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -109,88 +182,106 @@
 
   <ul>
     <li [ngClass]="{'active': currentUrl === '/learn/math'}">
-      <a (click)="toggleClassroomSubmenu()" class="protractor-mobile-test-classroom">
-        <i class="material-icons oppia-sidebar-menu-icon">school</i>
-        <span>{{ 'I18N_SIDEBAR_CLASSROOM' | translate }}</span>
-      </a>
-      <div [hidden]="!classroomSubmenuIsShown" class="classroom-submenu">
+      <div (click)="toggleClassroomSubmenuLearn()" class="oppia-sidebar-expandalbe-button protractor-mobile-test-learn" >
+        <i class="material-icons oppia-sidebar-menu-icon expand-icon expand-icon-learn oppia-sidebar-menu-icon-transition">expand_more</i>
+        <span class="bordertoggle-learn oppia-sidebar-submenu-toggle text-uppercase" >
+              {{ 'I18N_SIDEBAR_LEARN' | translate }}
+        </span>
+      </div>
+      <div [hidden]="!classroomSubmenuIsShownLEARN" class="oppia-sidebar-submenu-upperborder">
+        <div class="oppia-submenu-image-position">
+          <picture>
+            <source type="image/webp"
+                    [srcset]="this.getStaticImageUrl('/sidebar/learn-math-logo.webp')">
+            <source type="image/png"
+                    [srcset]="this.getStaticImageUrl('/sidebar/learn-math-logo.png')">
+            <img [src]="getStaticImageUrl('/sidebar/learn-math-logo.png')"
+                 class="oppia-submenu-image"
+                 alt="learn-math-logo">
+          </picture>
+        </div>
         <a href="/learn/math" routerLink="/learn/math" class="protractor-mobile-test-mathematics-link">
-          <span>{{ 'I18N_SIDEBAR_CLASSROOM_BASIC_MATHS' | translate }}</span>
+          <span>{{ 'I18N_SIDEBAR_MATH_FOUNDATIONS' | translate }}</span>
+          <P class="oppia-sidebar-learn-subtext">Beginner friendly lessons to help you get started in math.</P>
```

Comment:
Please add an I18n Key for the text and use translate pipe. Ditto else where. I18n keys help us to have text in all languages. so we try not to use text directly in html.

Reply:
done!